hhs_oauth_server/utils.py

```diff
@@ -1,9 +1,15 @@
+import logging
```

Comment:
Lets use the new logger here :) 
```suggestion
import apps.logging.request_logger as logging
```